bindings/wasm/bindings.cpp

```diff
@@ -141,6 +142,21 @@ Manifold Extrude(std::vector<std::vector<glm::vec2>>& polygons, float height,
                            twistDegrees, scaleTop);
 }
 
+std::vector<glm::ivec3> TriangulateJS(
+    std::vector<std::vector<glm::vec2>>& polygons) {
```

Comment:
Thanks, I was wondering how/when I should expose Triangulate as part of our public API. I'd like to keep our API consistent, so let's add this to the C++ and then expose it through JS, rather than writing it multiple times.

Reply:
I would propose in `polygon.h`, change `Triangulate()` to `TriangulateIdx()` and add a `Triangulate()` function that matches your signature.

Reply:
Good idea, I'll do the change in the following commit.